.gitignore

```diff
@@ -182,4 +182,7 @@ zenml_examples/
 !examples/**/*.yaml
 
 # stack recipes
-test_terraform/
\ No newline at end of file
+test_terraform/
+
+# static dashboard files
+src/zenml/zen_server/dashboard/
```

Comment:
Not sure if this interferes with how we build our package? I just added it into my `~/.gitignore_global` file. but maybe it's ok?

Reply:
@stefannica said it's fine, so should be good